src/paperless_tika/parsers.py

```diff
@@ -51,7 +53,9 @@ def parse(self, document_path: Path, mime_type: str, file_name=None):
 
         try:
             with TikaClient(tika_url=settings.TIKA_ENDPOINT) as client:
-                parsed = client.tika.as_text.from_file(document_path, mime_type)
+                with open(document_path, "rb") as f:
```

Comment:
Reading the whole file into memory isn't a valid solution to me.  The original issue shows Tika itself is crashing.  If it can't handle the document via a multi-part put, but can for a simple post, that's an issue inside tika.

So this change will need to be reverted